pkg/asset/quota/openstack/openstack.go

```diff
@@ -53,6 +53,13 @@ func Constraints(ci *validation.CloudInfo, controlPlanes []machineapi.Machine, c
 	if len(ci.ControlPlanePortSubnets) == 0 {
 		constraints = append(constraints, networkConstraint(1), routerConstraint(1), subnetConstraint(1))
 	}
+	// if the cluster does not have worker nodes then reduce the server group value from 2 to 1
+	numServerGroups := int64(2)
```

Comment:
Why do we need to wrap in `int64` rather than just using "raw" values? I'm guessing this is consistency, but it'd be nice to know why we did it in the first place :smile:

Reply:
If I remember correctly, the number of workers is int64 in install-config. The rest of the numbers can be automatic `int`s I believe

Reply:
I see that this is still int64. Was it not possible to use `int`?